articles/references/templates_decorators/the_template_class.md

```diff
@@ -2,17 +2,20 @@
 
 Templates are completely generic and can be implemented how ever you like, at it's core you simply have to implement this interface.
 
+>[!NOTE]
+>Though this is already managed for you by the [Module Builder](xref:ModuleBuilder), it is important for reference purposes to understand what these parts are about.
```

Comment:
As per one of my earlier comments, we could leave the square brackets empty to have the article name resolved automatically.